.github: Add initial target specifier config

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#39378 .github: Add initial target specifier config**

Will initially only contain a label to trigger builds for binary tests

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D21864091](https://our.internmc.facebook.com/intern/diff/D21864091)

seemethere committed:
.github: Add initial target specifier config

Will initially only contain a label to trigger builds for binary tests

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

malfet reviewed (approved):
Can you please rebase to make sure that all build failures are gone?

ezyang commented:
Would also be good to have a link to the documentation for the configuration format in the YAML file, for future readers.

seemethere committed:
Update on ".github: Add initial target specifier config"


Will initially only contain a label to trigger builds for binary tests

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere committed:
Update on ".github: Add initial target specifier config"


Will initially only contain a label to trigger builds for binary tests

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]
